plugins/oauth/hooks.js

```diff
@@ -62,6 +64,21 @@ module.exports = Object.assign({}, StoreMixin, {
     }]);
   },
 
+  dcosInstallCommand(instructions) {
+    let steps = instructions.split('&&');
+    steps.splice(-1, 0, ' \n dcos auth login ');
```

Comment:
I'd much rather duplicate code/data than having a hard to maintain and error prone implementation. DRY isn't absolute.